packages/devtools/lib/src/timeline/flame_chart_canvas.dart

```diff
@@ -612,16 +644,14 @@ class TimelineGrid {
       // Paint the timestamps and compute the line to stroke for the grid lines.
       canvas
         ..fillText(timestampText, timestampX, viewport.top + _textOffsetY)
+        ..beginPath()
         ..moveTo(left + currentInterval, visible.top)
-        ..lineTo(left + currentInterval, visible.bottom);
+        ..lineTo(left + currentInterval, visible.bottom)
+        ..closePath()
+        ..stroke();
```

Comment:
why did we switch to calling stroke every line segment instead of just at the end. was that actually faster?